### ISSUE TITLE
Unable to connect to quickbooks, stuck at "set up payments" page.

### ISSUE THREAD

**Issue by calvintwr** [n8n-io/n8n#1851]
**Describe the bug**
After creating the Quickbook credentials, when attempting to connect, the popup shows:

![Screenshot 2021-06-01 at 1 37 08 AM](https://user-images.githubusercontent.com/6825277/120226548-00ca7900-c27a-11eb-92bd-f457b3fea765.png)

My Quickbook account does not have payment set upt, and also, it is not available outside of US. Clicking on "No, thanks" results in the following error:

```
{"code":0,"message":"Insufficient parameters for OAuth2 callback. Received following query parameters: 
{\"error\":\"access_denied\",\"state\":\"eyJ0b2tlbiI6Ik9iQXh1cnkzLWU2a1JKVFZlZUo0Y3pBUE1DNGd6OVpBODNzYyIsImNp
ZCI6IjE0In0=\"}","stack":"ResponseError: Insufficient parameters for OAuth2 callback. Received following query 
parameters: 
{\"error\":\"access_denied\",\"state\":\"eyJ0b2tlbiI6Ik9iQXh1cnkzLWU2a1JKVFZlZUo0Y3pBUE1DNGd6OVpBODNzYyIsImNp
ZCI6IjE0In0=\"}\n    at /apps/n8n/node_modules/n8n/dist/src/Server.js:819:39\n    at Layer.handle [as handle_request] 
(/apps/n8n/node_modules/express/lib/router/layer.js:95:5)\n    at next 
(/apps/n8n/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch 
(/apps/n8n/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] 
(/apps/n8n/node_modules/express/lib/router/layer.js:95:5)\n    at 
/apps/n8n/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params 
(/apps/n8n/node_modules/express/lib/router/index.js:335:12)\n    at next 
(/apps/n8n/node_modules/express/lib/router/index.js:275:10)\n    at 
/apps/n8n/node_modules/n8n/dist/src/Server.js:259:13\n    at Layer.handle [as handle_request] 
(/apps/n8n/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix 
(/apps/n8n/node_modules/express/lib/router/index.js:317:13)\n    at 
/apps/n8n/node_modules/express/lib/router/index.js:284:7\n    at Function.process_params 
(/apps/n8n/node_modules/express/lib/router/index.js:335:12)\n    at next 
(/apps/n8n/node_modules/express/lib/router/index.js:275:10)\n    at 
/apps/n8n/node_modules/n8n/dist/src/Server.js:251:17\n    at Layer.handle [as handle_request] 
(/apps/n8n/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix 
(/apps/n8n/node_modules/express/lib/router/index.js:317:13)\n    at 
/apps/n8n/node_modules/express/lib/router/index.js:284:7\n    at Function.process_params 
(/apps/n8n/node_modules/express/lib/router/index.js:335:12)\n    at next 
(/apps/n8n/node_modules/express/lib/router/index.js:275:10)\n    at urlencodedParser (/apps/n8n/node_modules/body-
parser/lib/types/urlencoded.js:91:7)\n    at Layer.handle [as handle_request] 
(/apps/n8n/node_modules/express/lib/router/layer.js:95:5)"}

```

**To Reproduce**
1. Go to credentials.
2. Click on "new"
3. Select type "Quickbooks OAuth2"
3. Enter all credentials.
4. Click on the "connect" icon

**Expected behavior**
An Oauth page shows up allowing authorisation to continue

**Environment (please complete the following information):**
 - OS: MacOS 10.15.7
 - n8n Version 0.119.0
 - Node.js Version 16.1.0
 - Database system SQLite [default]
 - Operation mode N/A

**Comment by calvintwr** [comment]
i think it's because in:

packages/nodes-base/credentials/QuickBooksOAuth2Api.credentials.ts

```js
const scopes = [
	'com.intuit.quickbooks.accounting',
	'com.intuit.quickbooks.payment',
];
```
We need to create an option to make `'com.intuit.quickbooks.payment'` optional.

**Comment by janober** [comment]
Fix got merged with  #1859

**Comment by janober** [comment]
Got released with n8n@0.126.0
